tests/unit/scanners/test.javascript.js

```diff
@@ -419,6 +419,19 @@ describe('JavaScript Scanner', () => {
     ]);
   });
 
+  // See: https://github.com/mozilla/eslint-plugin-no-unsanitized/issues/188
+  it('has variable tracing disabled to avoid a bug', async () => {
+    const code = 'let c; a.innerHTML = `${c}`;';
+
+    const jsScanner = new JavaScriptScanner(code, 'code.js');
+
+    // We are not so much interested in the actual result. This would throw an
+    // error because of thebug in the upstream library.
```

Comment:
:ok_hand: as long as it visible fail when that options is turned on but not yet fixed in the dependency, I'm totally fine with it (on the contrary thanks for taking care of adding a smoke test :heart:).